tests/phpunit/unit/Configuration/ResourceManagerTest.php

```diff
@@ -178,9 +178,9 @@ public function testDefaultRequest()
         $app = new Application(['resources' => $config]);
         $this->assertEquals('cli', $config->getRequest('protocol'));
         $this->assertEquals('bolt.dev', $config->getRequest('hostname'));
-        $this->assertEquals('http://bolt.dev/bolt', $config->getUrl('canonical'));
+        $this->assertEquals('cli://bolt.dev/bolt', $config->getUrl('canonical'));
         $this->assertEquals('cli://bolt.dev', $config->getUrl('host'));
-        $this->assertEquals('http://bolt.dev/', $config->getUrl('rooturl'));
+        $this->assertEquals('cli://bolt.dev/', $config->getUrl('rooturl'));
```

Comment:
Not from my changes. Only `canonical` or `canonicalurl` is changed to point to `Canoincal` class while inside of request cycle. The whole class is deprecated anyways.